Patches/SQLite3/SQLite3Config.cmake.in

```diff
@@ -1,8 +1,8 @@
 # Config file used to import the SQLite3 build tree.
-set( SQLite3_INCLUDE_DIR "@CMAKE_INSTALL_PREFIX@/include" )
-set( SQLite3_LIBRARY sqlite3 )
-set( SQLite3_EXECUTABLE "@CMAKE_INSTALL_PREFIX@/sqlite3_exe" )
-set( EXE_SQLITE3 "@CMAKE_INSTALL_PREFIX@/bin/sqlite3_exe" )
+set( SQLite3_INCLUDE_DIR "@CMAKE_INSTALL_FULL_INCLUDEDIR@" )
```

Comment:
(With respect to the previous comment, I'll note that using `_FULL_` *here* is correct.)